plugin-server/tests/shared/process-event.ts

```diff
@@ -376,6 +380,7 @@ export const createProcessEventTests = (
             const chPeople = await hub.db.fetchPersons(Database.ClickHouse)
             expect(chPeople.length).toEqual(1)
             expect(JSON.parse(chPeople[0].properties)).toEqual(expectedProps)
+            expect(chPeople[0].created_at).toEqual(now.toFormat('yyyy-MM-dd HH:mm:ss.000'))
```

Comment:
nit: hmm not sure how readable this is

Reply:
it's not really, but I couldn't figure out a nicer way to do it.